NEWS.md

```diff
@@ -1,3 +1,7 @@
+# naomi 2.10.3
+
+* Return `anc_already_art`, `anc_status`, `anc_art_among_known` and `anc_total_pos` indicators from ANC input time series data.
```

Comment:
What is `anc_status`? Is that the total denominator?  I usually use `anc_hiv_status` for that. Doesn't matter too much